sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobContainerEncryptionScope.java

```diff
@@ -26,8 +26,8 @@
      * from specifying a different encryption scope than the scope set on the
      * container.
      */
-    @JsonProperty(value = "preventEncryptionScopeOverride")
-    private boolean preventEncryptionScopeOverride;
+    @JsonProperty(value = "encryptionScopeOverridePrevented")
```

Comment:
Was the JSON element name supposed to change?

Reply:
Shouldnt affect functionality